# Fix warnings catched by lintian: spelling, bare hyphen, unintended
# 'Cache-Control' expansion as macro
--- a/s3cmd.1	2011-01-09 10:03:06.000000000 +0100
+++ b/s3cmd.1	2011-03-28 12:24:06.441185593 +0200
@@ -386,7 +386,7 @@
 .SH AUTHOR
 Written by Michal Ludvig <mludvig@logix.net.nz> and 15+ contributors
 .SH CONTACT, SUPPORT
-Prefered way to get support is our mailing list:
+Preferred way to get support is our mailing list:
 .I s3tools\-general@lists.sourceforge.net
 .SH REPORTING BUGS
 Report bugs to 
